src/Server/TextDocument.php

```diff
@@ -411,4 +416,12 @@ public function xdefinition(TextDocumentIdentifier $textDocument, Position $posi
             return [new SymbolLocationInformation($descriptor, $def->symbolInformation->location)];
         });
     }
+
+    public function signatureHelp(TextDocumentIdentifier $textDocument, Position $position): Promise
```

Comment:
Please add a docblock for this method with the description from the protocol spec (makes it easier for newcomers to understand the purpose of the method)